### PR TITLE
feat: Add fecth api data and request handling

### PR DESCRIPTION
**Pull request by raulgomezc**
Closes [Ticket](https://trello.com)

## Testing

Steps to manually verify the change:
1. _Setup_: step to set the environment
2. _Exercise_: How to run the environment and example link: [Test Link](http://127.0.0.1:8000/)
3. _Verify_: what we need to verify

### CODE REVIEW

File: paradise_papers_search/fetch_api/models/helpers.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from neomodel import db, RelationshipManager
+from neomodel import db
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/models/helpers.py:1]: 👍

File: paradise_papers_search/fetch_api/models/helpers.py [hunk 2]
```diff
@@ -15,36 +15,103 @@
...
+    'ibcRUC',
+    'name',
+    'country_codes',
+    'incorporation_date',
+    'node_id',
+    'status',
+]
+
+# Serialize data to be JSON parsable
+### TODO: Refactor and impplement serializers on models
```
Comment (oscarpolanco) [inline paradise_papers_search/fetch_api/models/helpers.py:35]: 🙏

**Review by oscarpolanco — APPROVED** [review]
SGTM

File: paradise_papers_search/fetch_api/models/helpers.py (hunk 2 above)
Comment (junior1q94) [reply paradise_papers_search/fetch_api/models/helpers.py:35]: 📿

**Comment by junior1q94** [comment]
LGTM, waiting for the refactor PR.

**Review by junior1q94 — APPROVED** [review]

**Pull request merged**